frontend/src/lib/http/index.js

```diff
@@ -16,8 +16,12 @@ export const fetchX = async (url, options) => {
   const contentType = response.headers.get('Content-Type');
 
   if (contentType.includes('application/json')) {
-    const json = await response.json();
-    return json;
+    try {
+      const json = await response.json();
+      return json;
+    } catch (error) {
+      console.error(error);
+    }
```

Comment:
@bkach you might wanna remove the content type header from the endpoints that don't return anything, otherwise the JSON parsing fails. I'm swallowing it here but not an ideal solution. I think this is only for the game deletion endpoint. An alternative would be to return `{ "gameId": "" }`